target/social-map-1/CreateAnEvent.html

```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Create an Event</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="mapScript.js"></script>
+    <script src="profileScript.js"></script>
+    <script src="eventScript.js"></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+    <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyBf5E9ymEYBv6mAi78mFBOn8oUVvO8sph4&libraries=places"></script>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  </head>
+  <body onload="onload()">
+    <div class="navbar">
+      <p class="navbar-text" onclick="window.location.href = 'map.html'">Social Map</p>
```

Comment:
It's generally best to use anchor (`<a>`) tags for nav elements. They have some built-in accessibility features and don't rely on JavaScript to function.